ci/test.sh

```diff
@@ -6,8 +6,4 @@ if [ "$*" = "" ]; then
   set ./...
 fi
 
-if [ "${CI:-}" ]; then
```

Comment:
why remove this entirely? then other CI will fail

Reply:
It won't, it always go through `make.sh`.

Reply:
Oh but I guess I have to remove DAILY then.